src/test/java/net/revelc/code/impsort/FileTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.revelc.code.impsort;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.Test;
+
+/**
+ * Test class for special file cases.
+ */
+public class FileTest {
+
+  private static Grouper eclipseDefaults =
+      new Grouper("java.,javax.,org.,com.", "", false, false, true);
+
+  /**
+   * Test successfully parsing empty (0 byte) file.
+   */
+  @Test(expected = EmptyFileException.class)
+  public void test_empty_file() throws IOException, EmptyFileException, UnknownLineEndingException {
```

Comment:
```suggestion
  public void testEmptyFile() throws IOException, EmptyFileException, UnknownLineEndingException {
```

Reply:
There was another test that used this underscore naming although this maybe came from formatter `LineEdingTest`.

I had assumed you wrote it and that that was the preferred naming. Maybe you want to change that test as well. I can change all the methods to use camel case instead.

Reply:
I didn't write the other test, and hadn't noticed. It might have been another contributor. It's not a big deal either way. I wouldn't bother changing the other one, but it's probably best to stick to Java conventions for new stuff. If I happen across the other one and it really bothers me, I'll change the other one later. :smiley_cat: